data/systemd/snapd.manage

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+#
+# This script acts as an entry point for running snapd from snapd.snap on
+# a base snap that otherwise doesn't ship snapd itself. The entry point
+# arranges for appropriate system integration, in particular by creating
+# and starting systemd units for running snapd without any confinement.
+#
+# Arguments:
+# $1: start|stop
+# $2: prefix of the installed snapd snap (e.g. /snap/snapd/42/)
+#     (on a fedora base this might be e.g. /var/lib/snaps...)
+
+set -e
+
+CMD="$1"
+PREFIX="$2"
+
+UNITS="snapd.service snapd.autoimport.service snapd.core-fixup.service \
+       snapd.seeded.service snapd.snap-repair.service \
+       snapd.snap-repair.timer snapd.socket"
+ENABLE_UNITS="snapd.system-shutdown.service"
+
+# TODO: discuss ID_LIKE=ubuntu-core or ID_LIKE=snap-core ?
+if ! grep -q "ID=ubuntu-core" /etc/os-release && \
```

Comment:
One more point: /etc/os-release is optional. We should also check `/usr/lib/os-release`.

From the manual page:
```
       The file /etc/os-release takes precedence over /usr/lib/os-release.
       Applications should check for the former, and exclusively use its data
       if it exists, and only fall back to /usr/lib/os-release if it is
       missing. Applications should not read data from both files at the same
       time.  /usr/lib/os-release is the recommended place to store OS release
       information as part of vendor trees.  /etc/os-release should be a
       relative symlink to /usr/lib/os-release, to provide compatibility with
       applications only looking at /etc. A relative symlink instead of an
       absolute symlink is necessary to avoid breaking the link in a chroot or
       initrd environment such as dracut.
```

This would also aid our quest to make `/etc` empty.